samples/build_utils/CMakeLists.txt

```diff
@@ -213,7 +225,7 @@ macro(GetIGAHeaders TARGET)
                             ${IGA_INC_PATH}/IGA/iga_bxml_enums.hpp
                             ${IGA_INC_PATH}/IGA/kv.h
                             ${IGA_INC_PATH}/IGA/kv.hpp
-                    COMMAND python "${PROJECT_SOURCE_DIR}/../build_utils/get_iga_headers.py" ${IGA_INC_PATH} ${CMAKE_BINARY_DIR})
+                    COMMAND "${PYTHON_EXECUTABLE}" "${PROJECT_SOURCE_DIR}/../build_utils/get_iga_headers.py" ${IGA_INC_PATH} ${CMAKE_BINARY_DIR})
```

Comment:
Missing RequirePythonInterp() in this GetIGAHeaders macro

Reply:
Nice catch!